legacy/reqcounter/reqcounter_test.go

```diff
@@ -49,15 +49,17 @@ func TestInit(t *testing.T) {
 	require.True(t, rc.EnableCounting, "Init did not enable counting.")
 	// Ensure request counters are created.
 	require.Greater(t, len(rc.NetMonitor.RequestCounters), 0, "Init did not create any request counters within the global Monitor.")
-	// Ensure at least one request counter uses the MeasurementWindow.
-	found := false
+	// Ensure at least one request counter uses the QuotaWindowShort.
+	foundQuotaWindowShort, foundQuotaWindowLong := false, false
```

Comment:
Nit: typically I prefer to keep 1 statements per line (1 variable declaration, etc) unless there's an exceptionally good reason not to.